docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -12,18 +19,22 @@ export enum ChatItemTypes {
   divider,
 }
 
-interface ChatItem {
+interface ChatItemType {
   itemType: ChatItemTypes
 }
 
-interface ChatMessage extends ChatMessageProps, ChatItem {
+interface ChatMessage extends ChatMessageProps, ChatItemType {
   tabIndex: number
   'aria-labelledby': string
   text: string
 }
-interface Divider extends DividerProps, ChatItem {}
+interface Divider extends DividerProps, ChatItemType {}
 
-type ChatItemContentProps = ChatMessage | Divider
+type ChatItem = {
+  content?: ChatMessage | Divider
+  gutter?: AvatarProps
+  mine?: boolean
```

Comment:
Should we alter here the `gutterPosition` value depending on the mine property?

Reply:
done below